UML editor/src/main/java/codecain/CLI.java

```diff
@@ -225,19 +235,58 @@ - rename class Person Employee
      * @return the result of the add operation
      */
     private static String handleAddCommand(String[] tokens) {
-        if (tokens.length == 3 && tokens[1].equalsIgnoreCase("class")) {
-            return addClass(tokens[2]);
-        } else if (tokens.length == 4 && tokens[1].equalsIgnoreCase("relationship")) {
-            return addRelationship(tokens[2], tokens[3]);
-        } else if (tokens.length == 4 && tokens[1].equalsIgnoreCase("field")) {
-            return addField(tokens[2], tokens[3]);
-        } else if (tokens.length >= 5 && tokens[1].equalsIgnoreCase("method")) {
-            return addMethod(tokens[2], tokens[3], Arrays.copyOfRange(tokens, 4, tokens.length));
-        } else if (tokens.length == 4 && tokens[1].equalsIgnoreCase("parameter")) {
-            return addParameter(tokens[2], tokens[3], tokens[4]);
-        } else {
-            return "Invalid command. Use 'help' for available commands.";
+        if (tokens.length < 3) {
+            return "Usage: add <type> <name> [additional parameters]. Use 'help' for available commands.";
         }
+
```

Comment:
This type of formatting for the break statements, personally makes the code harder to read with the amount of white space there is on the screen.

If this could be reformatted to be more concise like the previous handling that would be appreciative on my end.